rca/static_src/sass/components/_section.scss

```diff
@@ -207,6 +215,22 @@
             }
         }
 
+        &--second-col\@medium {
```

Comment:
For the tabs, we want their notch wrapper to be of a different size between breakpoints. Fun!